openlibrary/core/follows.py

```diff
@@ -176,3 +176,16 @@ def most_followed(cls, limit=100):
             vars={'limit': limit},
         )
         return top_publishers
+
+    @classmethod
+    def is_following(cls, useranme):
+        oldb = db.get_db()
+        query = """
+            SELECT EXISTS(
+                SELECT 1
+                FROM follows
+                WHERE subscriber=$subscriber
+            )
```

Comment:
Quick reference to [`EXISTS` documentation](https://www.postgresql.org/docs/current/functions-subquery.html#FUNCTIONS-SUBQUERY-EXISTS), if needed.